src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandler.java

```diff
@@ -57,7 +67,8 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
         requireNonNull(callbackType, "callbacktype must not be null");
 
         return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
-                && callback.getEvent() == EventType.ACTION_POSTPONEMENT_REQUEST;
+                && callback.getEvent() == EventType.ACTION_POSTPONEMENT_REQUEST
+                && isScheduleListingEnabled;
```

Comment:
Just triple confirming that postponements only can be used for S&L cases

Reply:
Yes, Makeda confirmed this with the POs